.github/workflows/gempush.yml

```diff
@@ -0,0 +1,40 @@
+name: Ruby Gem
+
+on:
+  push:
+    branches: [ release-* ]
+
+jobs:
+  build:
+    name: Build + Publish
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        version: 2.6.x
+
+    - name: Publish to GPR
+      run: |
+        mkdir -p $HOME/.gem
+        touch $HOME/.gem/credentials
+        chmod 0600 $HOME/.gem/credentials
+        printf -- "---\n:github: Bearer ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+        gem build *.gemspec
+        gem push --KEY github --host https://rubygems.pkg.github.com/${OWNER} *.gem
+      env:
+        GEM_HOST_API_KEY: ${{secrets.GPR_AUTH_TOKEN}}
+        OWNER: username
```

Comment:
I can't seem to find documentation for this `username` value -- does it exist and I'm just not seeing it? Or do we need to pull the username out of [GITHUB_REPOSITORY](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables)?

Reply:
Oops. I think this probably just needs to be updated to the proper username? I think it's `hashicorp-vagrant`